sklearn/metrics/tests/test_classification.py

```diff
@@ -2882,6 +2883,25 @@ def test_classification_metric_division_by_zero_nan_validaton(scoring):
     cross_val_score(classifier, X, y, scoring=scoring, n_jobs=2, error_score="raise")
 
 
+def test_prfs_docstring_consistency():
+    """Check docstrings parameters of related metrics are consistent."""
+    pytest.importorskip(
+        "numpydoc",
+        reason="numpydoc is required to test the docstrings",
+    )
+    assert_docstring_consistency(
+        [
+            precision_recall_fscore_support,
+            f1_score,
+            fbeta_score,
+            precision_score,
+            recall_score,
+        ],
+        include_params=True,
+        exclude_params=["average", "zero_division"],
```

Comment:
I'd have a comment as why these are excluded

Reply:
Ah good pick up. Will note down. I think the differences in the docstrings for these items are required.